src/e2e/scala/temple/SimpleE2ETest.scala

```diff
@@ -21,15 +23,24 @@ class SimpleE2ETest extends FlatSpec with Matchers {
       ),
     )
 
-    // Only one folder should have been generated
-    Files.list(basePath).count() shouldBe 1
-    Files.exists(basePath.resolve("templeuser-db")) shouldBe true
+    // Two folders should have been generated
+    val expectedFolders = Set("templeuser-db", "templeuser").map(dir => basePath.resolve(dir))
+    Files.list(basePath).toScala(Set) shouldBe expectedFolders
 
-    // Only one file should be present in the user-db folder
+    // Only one file should be present in the templeuser-db folder
```

Comment:
This is the same as above: how about a utility function for checking a folder contains exactly a set of files?

Reply:
Not sure what this would look like? I don't like the idea of putting test logic in a utility function.

Reply:
I'm happy to update the others to follow this new style of checking though